src/RayTracer/Public/js/render-3d.js

```diff
@@ -0,0 +1,176 @@
+const $ = require('jquery');
+
+$(window).on('load', function () {
```

Comment:
This `window` is going to cause issues...we'll discuss this in person and find a solution.